modules/tinymce/changelog.txt

```diff
@@ -21,6 +21,8 @@ Version 5.5.0 (TBD)
     Fixed the `DOMUtils.split()` API incorrectly removing some content #TINY-6294
     Fixed pressing escape not focusing the editor when using multiple toolbars #TINY-6230
     Fixed the `dirty` flag not being correctly set during an `AddUndo` event #TINY-4707
+    Fixed `editor.selection.setCursorLocation` moving the cursor immediately outside pre elements in some circumstances #TINY-4058
```

Comment:
"immediately outside" - just wondering if a customer will understand that. @tylerkelly13 ?

Reply:
I'm not sure what that means... do you mean `editor.selection.setCursorLocation` was placing the cursor in-front of the `pre` block when instructed to place the cursor on content within the pre block?

Reply:
In this case, I found the bug by calling `setCursorLocation()` without any arguments (intended to place the cursor at the start of the document). But yes, it was meant to put the cursor on the content within the pre block, and instead placed it before the block.